src/g2.rs

```diff
@@ -415,6 +416,75 @@ impl G2Affine {
     pub const fn compressed_size() -> usize {
         COMPRESSED_SIZE
     }
+
+    fn read_raw_og<R: Read>(mut reader: R) -> Result<Self, std::io::Error> {
+        let mut buf = [0u8];
+        reader.read_exact(&mut buf)?;
+        let _infinity = buf[0] == 1;
```

Comment:
I'm guessing uncompressed uses one byte for identity, but just double checking. Doesn't blst use the free three bits for encoding identity and sign of `y`